CARDS-2725: Remove @ from CSV export headers - Update Serialization documentation

acrowthe opened the pull request:
Update serialization documentation with the new csv export options

acrowthe committed:
CARDS-2725: Add option to remove @ from CSV export headers
Update serialization documentation with the new csv export options

sdumitriu commented on doc/Serialization.md:
```diff
@@ -38,7 +38,7 @@ Multiple different export formats are supported. These include:
 
 #### Example: Weekly OAIP Form Export
-`/Questionnaires/OAIP.data.dataFilter:modifiedAfter=2025-07-19T02:00:00%5C.000-05:00.dataFilter:modifiedBefore=2025-07-26T02:00:00%5C.000-05:00.dataFilter:status=SUBMITTED.labels.formToSurveyLinks.csvIncludeFields:@survey=Survey.csvHeader:raw.questionnaireFilter:exclude=%252FQuestionnaires%252FOAIP%252Foaip_module1%252Foaip_visit_month.csv`
+`/Questionnaires/OAIP.data.dataFilter:modifiedAfter=2025-07-19T02:00:00%5C.000-05:00.dataFilter:modifiedBefore=2025-07-26T02:00:00%5C.000-05:00.dataFilter:status=SUBMITTED.labels.formToSurveyLinks.csvIncludeFields:@survey=Survey.csvHeader:raw.csvReplaceColumnIds:@=#.questionnaireFilter:exclude=%252FQuestionnaires%252FOAIP%252Foaip_module1%252Foaip_visit_month.csv`
```
Since this is a URL, it needs to be escaped.

sdumitriu committed:
CARDS-2725: Add option to remove @ from CSV export headers
Update serialization documentation with the new csv export options
Use proper URL-encoding in the URL
